docs/src/DeveloperDocumentation/changelog.md

```diff
@@ -0,0 +1,114 @@
+
+### Secondary Labels: Topic
+
+In addition to the release notes action labels, you can tag your PR with these following
+labels, and the release notes script will organize them appropriately:
+
+| Label                         | Changelog Category |
+|-------------------------------|--------------------|
+| release notes: highlight      | Highlights |
+| renaming                      | Items being renamed |
+| topic: algebraic geometry     | Changes related to Algebraic Geometry |
+| topic: combinatorics          | Changes related to Combinatorics |
+| topic: commutative algebra    | Changes related to Commutative Algebra |
+| topic: FTheoryTools           | Changes related to F-Theory Tools |
+| topic: groups                 | Changes related to Groups |
+| topic: LieTheory              | Changes related to Lie Theory |
+| topic: number theory          | Changes related to Number Theory |
+| topic: polyhedral geometry    | Changes related to Polyhedral Geometry |
+| topic: schemes                | Changes related to Schemes |
+| topic: toric schemes          | Changes related to Toric Schemes |
+| topic: toric varieties        | Changes related to Toric Varieties |
+| topic: tropical geometry      | Changes related to Tropical Geometry |
+| serialization                 | Changes related to serializing data in the MRDI file format ? |
+| enhancement                   | New features or extended functionality |
+| experimental                  | Only changes experimental parts of OSCAR |
+| optimization                  | Performance improvements or improved testing |
+| bug: crash                    | Fixed bugs that could lead to crashes |
+| bug                           | Other fixed bugs |
+| documentation                 | Improvements or additions to documentation |
+| package: AbstractAlgebra      | Changes related to the package AbstractAlgebra |
+| package: AlgebraicSolving     | Changes related to the package AlgebraicSolving |
+| package: GAP                  | Changes related to the package GAP |
+| package: Hecke                | Changes related to the package Hecke |
+| package: Nemo                 | Changes related to the package Nemo |
+| package: Polymake             | Changes related to the package Polymake |
+| package: Singular             | Changes related to the package Singular |
+
+
+## How to set labels when submitting or reviewing a PR
+
+Everyone submitting a PR should ideally follow this checklist if they also have the access
+rights for assigning labels. If they don't, they should ideally leave instructions that
+indicate which labels to set, and a PR reviewer should then perform the following steps.
+
+1. If your PR should *not* appear in the changelog, assign the `release notes: not needed` label and stop here.
```

Comment:
I'm not a native speaker but I think this could potentially be misunderstood. I'd suggest to rephrase this to something like this:
```suggestion
1. If your PR is *not* supposed to appear in the changelog, assign the `release notes: not needed` label and stop here.
```